pom.xml

```
@@ -80,6 +80,28 @@
             <artifactId>jackson-datatype-jsr310</artifactId>
             <version>2.13.0</version>
         </dependency>
+
+        <dependency>
+            <groupId>org.springframework.security</groupId>
+            <artifactId>spring-security-core</artifactId>
+            <version>5.5.0</version>
```

You can use a version variable that already defined in the pom.xml
```suggestion
            <version>${spring.security.version}</version>
```
Please fix similar cases below